go/teams/loader.go

```diff
@@ -305,11 +305,15 @@ func (l *TeamLoader) load2InnerLocked(ctx context.Context, arg load2ArgT) (res *
 }
 
 func (l *TeamLoader) load2InnerLockedRetry(ctx context.Context, arg load2ArgT) (*load2ResT, error) {
+	ctx, tbs := l.G().CTimeBuckets(ctx)
+	tracer := l.G().CTimeTracer(ctx, "TeamLoader.load2ILR", false)
```

Comment:
Should we think about gating this stuff out of prod somehow? Or is it too small to matter?